mailgun/listserv_client.py

```diff
@@ -221,6 +221,14 @@ def send_mail(self, from_address, reply_to_address, to_address, cc_address,
             'html': html
         }
 
+        # we accept a single address or a list of addresses in to_address.
+        # if it's a list, add in recipient_variables to make sure mailgun
+        # doesn't include the whole list in the to: field, per
+        #   https://documentation.mailgun.com/user_manual.html#batch-sending
+        if not isinstance(to_address, basestring):
```

Comment:
I don't think so...we're sending `cc_address` to mailgun as `'h:cc'`, so mailgun is generally going to ignore the addresses there anyway.

Maybe we want to rename the incoming arg to reflect that usage, but I can't for the life of me come up with a good rename.  You have any ideas?